[2025-05-01T09:00:34.079+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: lgbtq_model_pipeline.retrain manual__2025-05-01T09:00:14.295629+00:00 [queued]>
[2025-05-01T09:00:34.083+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: lgbtq_model_pipeline.retrain manual__2025-05-01T09:00:14.295629+00:00 [queued]>
[2025-05-01T09:00:34.084+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-05-01T09:00:34.091+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): retrain> on 2025-05-01 09:00:14.295629+00:00
[2025-05-01T09:00:34.096+0000] {standard_task_runner.py:57} INFO - Started process 241 to run task
[2025-05-01T09:00:34.099+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'lgbtq_model_pipeline', 'retrain', 'manual__2025-05-01T09:00:14.295629+00:00', '--job-id', '154', '--raw', '--subdir', 'DAGS_FOLDER/lgbtq_model_pipeline.py', '--cfg-path', '/tmp/tmpphtm12ux']
[2025-05-01T09:00:34.103+0000] {standard_task_runner.py:85} INFO - Job 154: Subtask retrain
[2025-05-01T09:00:34.127+0000] {task_command.py:416} INFO - Running <TaskInstance: lgbtq_model_pipeline.retrain manual__2025-05-01T09:00:14.295629+00:00 [running]> on host 09198680648f
[2025-05-01T09:00:34.166+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='lgbtq_model_pipeline' AIRFLOW_CTX_TASK_ID='retrain' AIRFLOW_CTX_EXECUTION_DATE='2025-05-01T09:00:14.295629+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-01T09:00:14.295629+00:00'
[2025-05-01T09:00:34.167+0000] {logging_mixin.py:154} INFO - üîÅ Starting retraining process...
[2025-05-01T09:00:34.168+0000] {logging_mixin.py:154} INFO - üì¶ Combining production and new data into all_data...
[2025-05-01T09:00:34.176+0000] {logging_mixin.py:154} INFO - üìä Loading all training data...
[2025-05-01T09:00:34.178+0000] {logging_mixin.py:154} INFO - üß† Retraining classifier on 5 samples...
[2025-05-01T09:00:34.178+0000] {logging_mixin.py:154} INFO - üìÇ Loading previous model from /opt/***/models/classifier.joblib
[2025-05-01T09:00:34.183+0000] {logging_mixin.py:154} WARNING - /home/***/.local/lib/python3.8/site-packages/sklearn/base.py:348 InconsistentVersionWarning: Trying to unpickle estimator LogisticRegression from version 1.5.1 when using version 1.3.2. This might lead to breaking code or invalid results. Use at your own risk. For more info please refer to:
https://scikit-learn.org/stable/model_persistence.html#security-maintainability-limitations
[2025-05-01T09:00:34.184+0000] {logging_mixin.py:154} INFO - üõ°Ô∏è Backed up previous model to /opt/***/models/classifier_backup_20250501090034.joblib
[2025-05-01T09:00:34.185+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/tasks/retrain.py", line 82, in run
    classifier.fit(X, y)
  File "/home/airflow/.local/lib/python3.8/site-packages/sklearn/base.py", line 1145, in wrapper
    estimator._validate_params()
  File "/home/airflow/.local/lib/python3.8/site-packages/sklearn/base.py", line 638, in _validate_params
    validate_parameter_constraints(
  File "/home/airflow/.local/lib/python3.8/site-packages/sklearn/utils/_param_validation.py", line 96, in validate_parameter_constraints
    raise InvalidParameterError(
sklearn.utils._param_validation.InvalidParameterError: The 'multi_class' parameter of LogisticRegression must be a str among {'multinomial', 'auto', 'ovr'}. Got 'deprecated' instead.
[2025-05-01T09:00:34.191+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=lgbtq_model_pipeline, task_id=retrain, execution_date=20250501T090014, start_date=20250501T090034, end_date=20250501T090034
[2025-05-01T09:00:34.196+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 154 for task retrain (The 'multi_class' parameter of LogisticRegression must be a str among {'multinomial', 'auto', 'ovr'}. Got 'deprecated' instead.; 241)
[2025-05-01T09:00:34.243+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-05-01T09:00:34.259+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
